aspnetcore/mvc/views/view-compilation.md

```diff
@@ -18,16 +18,17 @@ Razor files with a `.cshtml` extension are compiled at both build and publish ti
 > Runtime compilation:
 > * Isn't supported for Razor components of Blazor apps.
 > * Doesn't support [global using directives](/dotnet/csharp/whats-new/csharp-10#global-using-directives).
-> * Doesn't support [implicit using directives](/dotnet/core/tutorials/top-level-templates#implicit-using-directives)
+> * Doesn't support [implicit using directives](/dotnet/core/tutorials/top-level-templates#implicit-using-directives).
+> * Disables [.NET Hot Reload](xref:test/hot-reload).
 
 ## Razor compilation
 
 Build-time and publish-time compilation of Razor files is enabled by default by the Razor SDK. When enabled, runtime compilation complements build-time compilation, allowing Razor files to be updated if they're edited.
 
-In addition to build-time compilation, updating Razor views and Razor Pages is supported using <xref:test/hot-reload>.
+In addition to build-time compilation, updating Razor views and Razor Pages is supported using [.NET Hot Reload](xref:test/hot-reload).
 
 > [!NOTE]
-> When enabled, runtime compilation disables [.NET Hot Reload](xref:test/hot-reload).
+> When enabled, runtime compilation disables [.NET Hot Reload](xref:test/hot-reload). We recommend enabling runtime compilation only if recompiling is needed in production after deployment. Use .NET Hot Reload during development.
```

Comment:
Our guidance is actually the opposite. We recommend only using Razor runtime compilation during development and not in production. We also recommend using Hot Reload instead of Razor runtime compilation during development, but we understand that the development experience with hot reload still has some limitations when compared to Razor runtime compilation. We're working to improve hot reload so that Razor runtime compilation is no longer necessary with the intent to eventually stop supporting it. But we don't expect that to happen for .NET 8, so Razor runtime compilation will be with us for a while yet.